Assets/AppCenter/Plugins/AppCenterSDK/Auth/Android/AuthInternal.cs

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+#if UNITY_ANDROID && !UNITY_EDITOR
+using System;
+using System.Collections.Generic;
+using Microsoft.AppCenter.Unity.Internal.Utility;
+using UnityEngine;
+
+namespace Microsoft.AppCenter.Unity.Auth.Internal
+{
+    class AuthInternal
+    {
+        private static AndroidJavaClass _auth = new AndroidJavaClass("com.microsoft.appcenter.auth.Auth");
+
+        public static void PrepareEventHandlers()
+        {
+            AppCenterBehavior.InitializedAppCenterAndServices += PostInitialize;
+        }
+
+        private static void PostInitialize()
+        {
+            var instance = _auth.CallStatic<AndroidJavaObject>("getInstance");
+            AndroidJavaClass unityPlayer = new AndroidJavaClass("com.unity3d.player.UnityPlayer");
+            AndroidJavaObject activity = unityPlayer.GetStatic<AndroidJavaObject>("currentActivity");
+            instance.Call("onActivityResumed", activity);
+        }
+
+        public static void AddNativeType(List<IntPtr> nativeTypes)
+        {
+            nativeTypes.Add(AndroidJNI.FindClass("com/microsoft/appcenter/auth/Auth"));
```

Comment:
```suggestion
            nativeTypes.Add(_auth.GetRawClass());
```

Reply:
Can we keep the previous implementation? It is just for consistency, let's keep it same as others.